File: LayoutTests/http/tests/cache/resources/cacheable-iframe.php
<?php
require_once '../../resources/portabilityLayer.php';

$max_age = 12 * 31 * 24 * 60 * 60; //one year
$expires = gmdate(DATE_RFC1123, time() + $max_age);

header('Cache-Control: public, max-age=' . $max_age);
header('Expires: ' . $expires);
header('Content-Type: text/html');
exit();
?>

File: LayoutTests/http/tests/cache/resources/cacheable-iframe.py
#!/usr/bin/env python3

import sys
from datetime import datetime, timedelta

max_age = 12 * 31 * 24 * 60 * 60
expires = '{} +0000'.format((datetime.utcnow() + timedelta(seconds=max_age)).strftime('%a, %d %b %Y %H:%M:%S'))

sys.stdout.write(
    f'Cache-Control: public, max-age={max_age}\r\n'
    f'Expires: {expires}\r\n'
    'Content-Type: text/html\r\n\r\n'
)
sys.exit(0)